python/kfserving/kfserving/server.py

```diff
@@ -65,6 +81,49 @@ def register_model(self, model):
         self.registered_models[model.name] = model
 
 
+class ModelPredictHandler(tornado.web.RequestHandler):
+    def initialize(self, protocol_handler, models):
+        self.models = models
+        self.protocol_handler = protocol_handler
```

Comment:
>protocol_handler [](start = 32, length = 16)

is the intent that the protocol is pluggable and i can use either seldon or tf wireformats with my model foo? i see the seldon handler converts to python types, but the tf handler is just passthrough..

Reply:
This needs further thought for the tensorflow protocol so that we provide a standard data payload to the underlying models. At present the seldon part converts to numpy array. I assume we could agree on that?

Reply:
Updated to place logic in servers - see XGBServer.